iamproperty/iamproperty.github.io
# Banner component

## Summary of Changes:
1. Banner/promo component

## Rationale
Banner/promo component built up if utility classes.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/banner

## Ticket(s)
DS-56

## Checklist

## Notes
